ASB Dec 2025 Security Patches integration

alavalas1 opened the pull request:
Integrating Google Android Security Bulletin Patches

Test done: STS r45 TCs Passed except 14 Test and
CtsSecurityTestCases Module
Screen lock issue (Issue Id: OAM-134026)
android.security.sts.KernelLtsTest#testRequiredKernelLts_WARN (Issue ID: OAM-133684)
CtsSecurityTestCases Module ( Issue Id:OAM-134426)

Tracked-On: OAM-134643

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

alavalas1 committed:
ASB Dec 2025 Security Patches integration

Integrating Google Android Security Bulletin Patches

Test done: STS r45 TCs Passed except 14 Test and
CtsSecurityTestCases Module
Screen lock issue (Issue Id: OAM-134026)
android.security.sts.KernelLtsTest#testRequiredKernelLts_WARN
(Issue ID: OAM-133684)
CtsSecurityTestCases Module ( Issue Id:OAM-134426)

Tracked-On: OAM-134643
Signed-off-by: alavala srinivasa reddy <alavala.srinivasa.reddy@intel.com>

sysopenci commented:
FAILURE: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

shyjumon-n reviewed (approved):
LGTM

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /android-ci-local/celadon_umr0_master-merge/517
